packages/ag-charts-community/src/chart/themes/chartTheme.ts

```diff
@@ -135,31 +135,31 @@ export class ChartTheme {
                 enabled: false,
                 text: 'Axis Title',
                 spacing: 25,
-                fontWeight: { ref: 'fontWeight' },
-                fontSize: FONT_SIZE.MEDIUM,
-                fontFamily: { ref: 'fontFamily' },
-                color: { ref: 'foregroundColor' },
+                fontWeight: { $ref: 'fontWeight' },
+                fontSize: { $round: [{ $mul: [{ $ref: 'fontSize' }, FONT_SIZE.MEDIUM / FONT_SIZE.SMALL] }] },
```

Comment:
These font sizes are unchanged for the default size, but scale at the same ratio.